regimes/it/scenarios.go

```diff
@@ -17,89 +80,141 @@ var invoiceScenarios = &tax.ScenarioSet{
 		// **** TIPO DOCUMENTO ****
 		{
 			Types: []cbc.Key{bill.InvoiceTypeStandard},
-			Meta: cbc.Meta{
+			Name: i18n.String{
+				i18n.EN: "Regular Invoice",
+				i18n.IT: "Fattura",
+			},
+			Codes: cbc.CodeSet{
```

Comment:
This makes sense for Italy, I just hope it translates well to other regimes in the future.